lib/slack-profile-cache.js

```diff
@@ -251,3 +252,18 @@ function timeout (ms) {
 async function sleep (timeoutMs) {
   await timeout(timeoutMs)
 }
+
+async function getUsersList (users = [], cursor = '') {
+  try {
+    const res = await slackWebClient.users.list({limit: 200, cursor: cursor})
+    users = users.concat(res.members)
+
+    if (!res.response_metadata.next_cursor) {
+      return users
+    }
+
+    return getUsersList(users, res.response_metadata.next_cursor)
+  } catch (e) {
```

Comment:
In this case, there's no point in catching the exception, we can just get rid of the `try/catch`